src/main/java/net/projectzombie/region_rotation/main/Main.java

```diff
@@ -20,12 +21,19 @@
  */
 public class Main extends JavaPlugin
 {
+    static private JavaPlugin PLUGIN;
+    static private Plugin WG_PLUGIN;
+
+    static public JavaPlugin plugin()
+    { return PLUGIN; }
+
+    static public Plugin WGplugin()
+    { return WG_PLUGIN; }
+
     @Override
     public void onEnable()
     {
-        // Start up read ins and sorts.
-        FileBufferController.init(this);
-        StateController.init(this);
+        StateController.init();
```

Comment:
PLUGIN = this;
WG_PLUGIN = WGBukkit.getPlugin();
Needs to be before the statecontroller gets init.